src/Sentry.Unity.Editor.iOS/SentryXcodeProject.cs

```diff
@@ -17,9 +16,31 @@ internal class SentryXcodeProject : IDisposable
 
         private readonly string _mainPath = Path.Combine("MainApp", "main.mm");
         private readonly string _optionsPath = Path.Combine("MainApp", OptionsName);
+        private readonly string _uploadScript = $@"process_upload_symbols()
+{{
+    ERROR=$(./{SentryCli.SentryCliMacOS} upload-dif $BUILT_PRODUCTS_DIR > ./sentry-symbols-upload.log 2>&1 &)
+    if [ ! $? -eq 0 ] ; then
+        echo ""warning: sentry-cli - $ERROR""
+    fi
+}}
+
+export SENTRY_PROPERTIES=sentry.properties
+if [ ""$ENABLE_BITCODE"" = ""NO"" ] ; then
+    echo ""Bitcode is disabled""
+    echo ""Uploading symbols""
+    process_upload_symbols
+else
+    echo ""Bitcode is enabled""
+    if [ ""$ACTION"" = ""install"" ] ; then
+        echo ""Uploading symbols and bcsymbolmaps""
+        process_upload_symbols
+    else
+        echo ""Skipping symbol upload on bitcode enabled and non-install builds""
+    fi
+fi";
 
         private readonly string _projectRoot;
-        private readonly PBXProject _project;
+        private readonly object _project = null!;
```

Comment:
Sounds like you only need the type:

```suggestion
        private readonly Type _pbxProjectType = null!; // Set in the ctor or throws
```